### PR TITLE
new_audit(seo): manual SEO audits

### LINKED ISSUE #3119: [SEO Audits] Manual SEO validation group
**Group name:** Manual SEO Validation

**Description:**  Run these additional validators on your site to check additional SEO best practices.

**Manual Audits:**
- [ ] [Page is mobile friendly](https://github.com/GoogleChrome/lighthouse/issues/3183)
- [ ] [Structured data is valid](https://github.com/GoogleChrome/lighthouse/issues/3184)

### LINKED ISSUE #3183: [SEO Audits] Page is mobile friendly
**Audit group:** [Manual SEO validation](https://github.com/GoogleChrome/lighthouse/issues/3119)
**Description:** Page is mobile friendly
**Help text:** The web is being accessed more and more on mobile devices. Designing your websites to be mobile friendly ensures that your pages perform well on all devices. Take the [Mobile-Friendly Test](https://search.google.com/test/mobile-friendly) to see how easily a visitor can use your page on a mobile device.

### LINKED ISSUE #3184: [SEO Audits] Structured data is valid
**Audit group:** [Manual SEO validation](https://github.com/GoogleChrome/lighthouse/issues/3119)
**Description:** Structured data is valid
**Help text:** Structured data is a standardized format for providing information about a page and classifying the page content; for example, if is it a recipe page, what are the ingredients, the cooking time and temperature, the calories, and so on. Run the [Structured Data Testing Tool](https://search.google.com/structured-data/testing-tool/) and the [Structured Data Linter](http://linter.structured-data.org/) to validate structured data usage.

### PR DESCRIPTION
**Pull request by kdzwinel**
Adds two manual SEO audits.

screen shot 2017-12-22 at 02 27 23

Closes #3119
Closes #3183
Closes #3184

### CODE REVIEW

File: lighthouse-core/config/default.js [hunk 1]
```diff
@@ -241,6 +243,10 @@ module.exports = {
       title: 'Crawling and Indexing',
       description: 'To appear in search results, crawlers need access to your app.',
     },
+    'manual-seo-checks': {
+      title: 'Manual SEO Validation',
```
Comment (kdzwinel) [inline lighthouse-core/config/default.js]: Both accessibility and PWA categories call this subcategory 'Additional items to manually check'  - IMO we should have the same name here.

Comment (patrickhulce) [reply lighthouse-core/config/default.js]: +1 :)

File: lighthouse-core/audits/seo/manual/mobile-friendly.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+ */
+
+class MobileFriendly extends ManualAudit {
+  /**
+   * @return {!AuditMeta}
+   */
+  static get meta() {
+    return Object.assign({
+      name: 'mobile-friendly',
+      helpText: 'Take the [Mobile-Friendly Test](https://search.google.com/test/mobile-friendly) to see how easily a visitor can use your page on a mobile device. [Learn more](https://developers.google.com/search/mobile-sites/).',
```
Comment (patrickhulce) [inline lighthouse-core/audits/seo/manual/mobile-friendly.js]: given that LH is all about mobile web advice, I'm assuming the plan is to convert this into an automatic test at some point? :)

File: lighthouse-core/config/default.js (hunk 1 above)
Comment (rviscomi) [reply lighthouse-core/config/default.js]: 👍

**Comment by kdzwinel** [comment]
Done 👌

screen shot 2018-01-03 at 02 10 27

Thank you @patrickhulce & @rviscomi for a review!

**Comment by paulirish** [comment]
lgtm from me. @rviscomi anything from you?

File: lighthouse-core/audits/seo/manual/mobile-friendly.js (hunk 2 above)
Comment (paulirish) [reply lighthouse-core/audits/seo/manual/mobile-friendly.js]: As discussed, this is currently using these 6 rules in the check: https://support.google.com/webmasters/answer/6352293?utm_source=search_console&utm_campaign=sc-mft-test

once we have coverage for all of them in LH, we can migrate this to a multicheckaudit and drop the recommendation to use MFT as we'll be testing the exact same thing. (we could also validate there is no significant difference in results across x000 URLs)

**Pull request merged**